docs/openapi.yaml

```diff
@@ -719,7 +719,7 @@ paths:
                 id:
                   type: string
                   pattern: '^[0-9a-zA-Z_-]{43}$'
-                  description: TX ID of the transaction you would like to prioritize.
+                  description: TX ID of the transaction or bundle you would like to prioritize.
```

Comment:
We should keep the original wording on this one. It must be a TX ID. An ID for a nested bundle wouldn't work.